Anastasia: homework_1 done

AnastasiaArts opened the pull request:
anastasia: homework 1 completed

Commit:
homework_1 done

Commit:
Fixes for Lint review
Files: task_1, task_2, task_3, task_4, task_5:
1. added newline at end of file
2. added whitespace around arithmetic operators

eokulik reviewed (changes requested):
Мы ж тут на автоматизаторов учимся - поэтому первая проверка - автоматическая )) Его имя линтер. И он тобой не очень доволен ))
Результат его работы находится на вкладке Checks, смотреть нужно на то, что он пишет под строкой `/opt/hostedtoolcache/Python/3.10.8/x64/bin/flake8 --ignore F401,W503 --max-line-length 120 .` Там написано в каком файле чо не так и в какой строке.

Commit:
Fixes for Lint review
Files: task_1, task_2, task_3, task_4, task_5:
1. removed variable 'e' for Exceptions as assigned to but never used

AnastasiaArts reviewed (commented):
@eokulik did you remove empty lines that were requested by Lint?

eokulik reviewed (approved):
Есть вопросики к циклам и перехвату эксепшенов, но к тому, что касается части выполнения домашки - нет
